app/templates/gulpfile.js

```diff
@@ -293,61 +294,64 @@ gulp.task('optimize', ['inject', 'test'], function() {
     // Replace the file names in the html with rev numbers
     .pipe($.revReplace())
     .pipe(gulp.dest(config.build));
+
+    combined.on('error', console.error.bind(console));
+  return combined;
 });
 
 /**
  * Remove all files from the build, temp, and reports folders
  * @param  {Function} done - callback when complete
  */
-gulp.task('clean', ['clean-fonts'], function(done) {
+gulp.task('clean', ['clean-fonts'], function() {
   var delconfig = [].concat(config.build, config.temp, config.report);
   log('Cleaning: ' + $.util.colors.blue(delconfig));
-  del(delconfig, done);
+  return del(delconfig);
```

Comment:
Returning stream cleaner/easier than working with the callback, which from the looks of it, wasn't being called at all. See also https://github.com/gulpjs/gulp/blob/master/docs/API.md#async-task-support